server.go

```diff
@@ -260,6 +301,11 @@ func loadState() (*grange.State, int) {
 	return &state, warnings
 }
 
+func cleanupStatsd() {
+	stats.Close()
+	stats = &noopStatsd{}
```

Comment:
Hygeine, maybe being too paranoid. I didn't want to have a closed stats client hanging around and maybe accidentally being used. (This method also gets called in a `defer` from main.)